docs/best-practices.md

```diff
@@ -0,0 +1,101 @@
+# Best Practices
+
+The items and criteria of flexibility in what is a good thing to do. Sections should offer contradictions of things that are commonly done.
+
+## React Coding
+
+### Avoid Custom Styling
```

Comment:
@jrenee42 we had this convo once haha -- just an FYI for when you get back... I guess this would be our official documentation on the statement of "no css"